testgrid/config.yaml

```diff
@@ -4645,6 +4629,27 @@ dashboards:
   - name: packages-pushed-master
     test_group_name: periodic-kubernetes-e2e-packages-pushed
 
+- name: sig-release-master-informing
```

Comment:
💯 @spiffxp this is somewhere in that wall of text about conformance I sent you the other day 😅